roles/cs.varnish/templates/vcl/subroutines/recv.vcl.j2

```diff
@@ -240,6 +240,8 @@ if (req.http.Accept-Encoding) {
     }
 }
 
+# Copy req.url to separate header before cut
+set req.http.X-MageOps-Full-URL = req.url;
```

Comment:
Please allow this header to be configured in defaults.yml
Also in this case I would stick to some more standard naming like `X-Original-URL`